elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/SubCountry.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.example;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.datastores.aggregation.annotation.Cardinality;
+import com.yahoo.elide.datastores.aggregation.annotation.CardinalitySize;
+import com.yahoo.elide.datastores.aggregation.annotation.FriendlyName;
+
+import org.hibernate.annotations.Subselect;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+/**
+ * A root level entity for testing AggregationDataStore with @Subselect annotation
+ */
+@Data
+@Entity
+@Include(rootLevel = true)
+@Subselect(value = "select * from countries")
+@Cardinality(size = CardinalitySize.SMALL)
+public class SubCountry {
```

Comment:
I'm not aware of a concept of a SubCountry.  It would be nice if we kept the test model representing something that is meaningful and fits the overall model example.  Let's find something else to model this use case.